source/AI.h

```diff
@@ -180,6 +185,7 @@ template <class Type>
 
 private:
 	// Data from the game engine.
+	const PlayerInfo &player;
```

Comment:
@tehhowch I'm having a bit of trouble trying to avoid the `player` field. I would need to pass around 3 sets (seen systems, visited systems, visited planets) around to avoid using `player`, which isn't that great imo.

So, what would you recommend?